scripts/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.3"
+services:
+  hello:
+    image: busybox
+    command:
+      sh -c 'while true; do nc -lp 8000 -e echo -e "HTTP/1.1 200 OK\r\n\r\n{\"msg\":\"hello\",\"host\":\"$$HOSTNAME\"}"; done'
```

Comment:
a tak z ciekawości jaki będzie rezultat tej komendy?

Reply:
W pętli będzie odpalał netcat'a słuchając na 8000, i na każdy request odpowiadał HTTP:OK.

Reply:
Prawda to, proszę o resolve jeśli temat wyczerpany ;)